environment.yml

```diff
@@ -414,67 +414,15 @@ dependencies:
   - zope.interface=5.1.0=py37h8f50634_0
   - zstd=1.4.5=h6597ccf_2
   - pip:
-    - aiohttp==3.6.3
-    - aiohttp-cors==0.7.0
-    - aioredis==1.3.1
-    - astunparse==1.6.3
-    - async-timeout==3.0.1
-    - blessings==1.7
-    - cachetools==4.1.1
-    - colorful==0.5.4
-    - dill==0.3.2
-    - dm-tree==0.1.5
-    - en-core-web-sm==2.3.1
-    - flax==0.2.2
-    - gast==0.3.3
-    - google==3.0.0
-    - google-api-core==1.22.4
-    - google-auth==1.22.1
-    - google-auth-oauthlib==0.4.1
-    - google-pasta==0.2.0
-    - googleapis-common-protos==1.52.0
-    - gpustat==0.6.0
-    - grpcio==1.32.0
-    - hiredis==1.1.0
-    - importlib-resources==3.0.0
-    - jax==0.2.3
-    - jaxlib==0.1.55
-    - keras-preprocessing==1.1.2
-    - markdown==3.3.1
-    - modin==0.8.1.1
-    - multidict==4.7.6
-    - numpy==1.18.5
-    - nvidia-ml-py3==7.352.0
-    - oauthlib==3.1.0
-    - opencensus==0.7.11
-    - opencensus-context==0.1.2
-    - opt-einsum==3.3.0
-    - promise==2.3
-    - protobuf==3.13.0
-    - py-spy==0.3.3
-    - pyarrow==0.16.0
-    - pyasn1==0.4.8
-    - pyasn1-modules==0.2.8
     - pyqt5-sip==4.19.18
     - pyqtchart==5.12
     - pyqtwebengine==5.12.1
-    - ray==1.0.0
-    - redis==3.4.1
-    - requests-oauthlib==1.3.0
-    - rsa==4.6
     - s2geometry==0.9.0
     - sacremoses==0.0.43
-    - sentencepiece==0.1.91
-    - tensorboard==2.3.0
-    - tensorboard-plugin-wit==1.7.0
-    - tensorflow-datasets==4.0.1
-    - tensorflow-estimator==2.3.0
-    - tensorflow-gpu==2.3.1
-    - tensorflow-metadata==0.24.0
-    - termcolor==1.1.0
-    - tokenizers==0.8.1rc2
-    - torchtext==0.7.0
-    - transformers==3.3.1
+    - tokenizers==0.9.4
+    - torch==1.7.1
+    - torchtext==0.8.1
+    - transformers==4.1.1
     - webcolors==1.11.1
-    - yarl==1.5.1
-prefix: /opt/conda/envs/cabby
+prefix: /home/nlp/tzufar/anaconda3/envs/cabby
```

Comment:
Can you change this prefix back?

Reply:
Thanks, didn't notice that. Changed it back to the previous prefix.